### ISSUE TITLE
DialogueTree::LineText() incompatible with DialogueTree::ClippedLineText()

### ISSUE THREAD

**Issue by PascalLadalle** [4ian/GDevelop#1631]
Not sure if this should go here, in feature requests or on the wiki. 😅 
Using DialogueTree::LineText() stops the scrolling.
Scrolling works or not, depending on whether I disable or not this event:
![image](https://user-images.githubusercontent.com/25978830/78720759-8bd60a80-7926-11ea-9015-de611c20a54c.png)

**Comment by blurymind** [comment]
Yeah, unfortunately I think when you get the line text, it also triggers the scrolling text to complete. 

In your case this is a problem because you don't want it to affect the condition that catches when it completes scrolling
